build.gradle.kts

```diff
@@ -0,0 +1,41 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm").apply(false)
+}
+
+
+allprojects {
+    group = "playground"
+
+    repositories {
+        mavenLocal()
+        mavenCentral()
+        google()
+        jcenter()
+        maven("https://dl.bintray.com/kotlin/kotlin-eap/")
+        maven("https://kotlin.bintray.com/kotlinx/")
+    }
+
+    tasks.withType<KotlinCompile> {
+        kotlinOptions.jvmTarget = "1.8"
+        kotlinOptions.freeCompilerArgs = listOf(
+            "-Xopt-in=kotlin.RequiresOptIn"
+        )
+    }
+}
+
+/**
+ * How do I setup GitHub Actions for my Gradle or Android project?
+ * https://dev.to/jmfayard/how-do-i-setup-github-actions-for-my-gradle-or-android-project-3eal
+ */
+tasks.register("runOnGitHub") {
+    dependsOn(":kotlin-jvm:run")
+    group = "custom"
+    description = "$ ./gradlew runOnGitHub # runs on GitHub Action"
+}
+
+tasks.register<DefaultTask>("hello") {
+    group = "Custom"
+    description = "Minimal task that do nothing. Useful to debug a failing build"
+}
```

Comment:
We don't need it, we can just use the existing `help` task.

Reply:
./gradlew hello is better, less noise, it's a common pattern in the Gradle docs